python/note_input_widget/widget.py

```diff
@@ -341,79 +341,88 @@ def _async_submit_note(self, sg, data):
-            sg_entity = sg.find_one(entity_link["type"], 
-                                    [["id", "is", entity_link["id"] ]], 
+            sg_entity = sg.find_one(entity_link["type"],
+                                    [["id", "is", entity_link["id"] ]],
                                     ["cached_display_name"])
-            note_links += [{"id": entity_link["id"], 
-                           "type": entity_link["type"], 
-                           "name": sg_entity["cached_display_name"] }] 
-            
-        
+            note_links += [{"id": entity_link["id"],
+                           "type": entity_link["type"],
+                           "name": sg_entity["cached_display_name"] }]
+
+
         elif entity_link["type"] == "Version":
-            # if we are adding a note to a version, link it with the version 
+            # if we are adding a note to a version, link it with the version
             # and the entity that the version is linked to.
-            # if the version has a task, link the task to the note too.  
-            sg_version = sg.find_one("Version", 
-                                     [["id", "is", entity_link["id"] ]], 
+            # if the version has a task, link the task to the note too.
+            sg_version = sg.find_one("Version",
+                [["id", "is", entity_link["id"] ]],
                                      ["entity", "sg_task", "cached_display_name", "project"])
-            
+
             # first make a std sg link to the current entity - this to ensure we have a name key present
-            note_links += [{"id": entity_link["id"], 
-                            "type": entity_link["type"], 
-                            "name": sg_version["cached_display_name"] }] 
-            
+            note_links += [{"id": entity_link["id"],
+                            "type": entity_link["type"],
+                            "name": sg_version["cached_display_name"] }]
+
             # and now add the linked entity, if there is one
             if sg_version["entity"]:
                 note_links += [sg_version["entity"]]
-            
+
             if sg_version["sg_task"]:
                 note_tasks += [sg_version["sg_task"]]
 
             # If we weren't able to get a project ID from the context, then
             # we know we can get it from the Version entity itself.
             if not project and sg_version["project"]:
                 project = sg_version["project"]
-            
+
         elif entity_link["type"] == "Task":
             # if we are adding a note to a task, link the note to the entity that is linked to the
             # task. The link the task to the note via the task link.
-            sg_task = sg.find_one("Task", 
-                                  [["id", "is", entity_link["id"] ]], 
+            sg_task = sg.find_one("Task",
+                [["id", "is", entity_link["id"] ]],
                                   ["entity", "project"])
```

Comment:
i think this may have happened as i was merging in some changes from master... fixed now.